course/models/models.py

```diff
@@ -175,6 +184,10 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
         ensure_uqj(None, self)
 
+    def delete(self):
```

Comment:
This is bad! We won't be able to delete questions from admin either!
This logic should be in the API only.

Reply:
Whoops right!